<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>VplotR package • VplotR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/sandstone/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="VplotR package">
<meta property="og:description" content="The pattern of digestion and protection from DNA nucleases such 
    as DNAse I, micrococcal nuclease, and Tn5 transposase can be used to 
    infer the location of associated proteins. This package contains useful 
    functions to analyze patterns of paired-end sequencing fragment density. 
    VplotR facilitates the generation of V-plots and footprint profiles 
    over single or aggregated genomic loci of interest.">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">VplotR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.12.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="articles/VplotR.html">Get started</a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
<li>
  <a href="news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/js2264/VplotR/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">


<div class="section level1">
<div class="page-header"><h1 id="vplotr-">VplotR <img src="inst/figures/logo.png" align="right" alt=""><a class="anchor" aria-label="anchor" href="#vplotr-"></a>
</h1></div>
<div class="section level2">
<h2 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<p>This R package makes the process of generating fragment density plots (also known as “V-plots”) straightforward. V-plots have been introduced <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3215028/" class="external-link">for the first time</a> by the Henikoff lab in 2011. Recently, V-plots have proven to be very instructive to understand the molecular organization of the chromatin. For instance, the <a href="https://genome.cshlp.org/content/early/2015/08/27/gr.192294.115" class="external-link">nucleoATAC</a> package relies on cross-correlation of ATAC-seq fragment density plots to accurately map nucleosome occupancy along the genome.<br>
VplotR aim is to streamline the process of generating V-plots. It contains wrapping functions to import paired-end sequencing bam files and generate V-plots around genomic loci of interest.<br>
VplotR is designed around <a href="https://ggplot2.tidyverse.org/" class="external-link">ggplot2</a> and makes full use of its potential. As such, it is easy to generate V-plots in batches and combine them with other plots to make publication-ready figures.</p>
</div>
<div class="section level2">
<h2 id="vplotr-documentation">VplotR documentation<a class="anchor" aria-label="anchor" href="#vplotr-documentation"></a>
</h2>
<p>VplotR documentation is available <a href="https://jserizay.com/VplotR/articles/VplotR.html" class="external-link">here</a> or directly within R:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"VplotR"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="v-plots-what-they-are-and-how-to-interpret-them">V-plots: what they are and how to interpret them<a class="anchor" aria-label="anchor" href="#v-plots-what-they-are-and-how-to-interpret-them"></a>
</h2>
<p>V-plots are two-dimensional plots of paired-end sequencing fragments from chromatin accessibility assays (e.g. ATAC-seq, MNase-seq or DNAse-seq) of from chromatin immunoprecipitation assays (i.e. ChIP-seq).</p>
<p>The x axis represents the distance between sequencing fragments and genomic loci of interest (e.g. nucleosome-depleted regions, or NDR), while the y axis indicates the length of the sequenced fragments. The color scale indicates the density of fragments mapping over aggregated genomic loci of interest.</p>
<div class="float">
<img src="inst/figures/concept.png" alt="concept"><div class="figcaption">concept</div>
</div>
<p>Thus, V-plots can reveal where protein bound to DNA (e.g. nucleosomes) protect it from digestion.</p>
</div>
<div class="section level2">
<h2 id="system-requirements">System requirements<a class="anchor" aria-label="anchor" href="#system-requirements"></a>
</h2>
<p>VplotR has been tested on Mac OS (<code>&gt;= 10.13</code>), Ubuntu (<code>&gt;= 18.04</code>) and Windows Server (<code>2012</code>).<br>
VplotR requires <code>R &gt;= 4.0</code> and is available from Bioconductor (release <code>&gt; 3.12</code>).</p>
</div>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>VplotR and all its dependencies can be installed from Bioconductor:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"VplotR"</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://github.com/js2264/VplotR" class="external-link">"VplotR"</a></span><span class="op">)</span></span></code></pre></div>
<p>VplotR developpment version can be installed from Github as follows:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"devtools"</span><span class="op">)</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"js2264/VplotR"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="citation">Citation<a class="anchor" aria-label="anchor" href="#citation"></a>
</h2>
<p>If you are using VplotR in your research, please cite:</p>
<blockquote>
<p><a href="https://joss.theoj.org/papers/10.21105/joss.03009" class="external-link">Generating fragment density plots in R/Bioconductor with VplotR</a><br><strong>J. Serizay</strong> &amp; J. Ahringer<br>
Journal of Open Source Software, 2021</p>
</blockquote>
<blockquote>
<p><a href="https://genome.cshlp.org/content/early/2020/10/21/gr.265934.120.abstract" class="external-link">Distinctive regulatory architectures of germline-active and somatic genes in <em>C. elegans</em></a><br><strong>J. Serizay</strong>, Y. Dong, J. Jänes, M. Chesney, C. Cerrato &amp; J. Ahringer<br>
Genome Research, 2020</p>
</blockquote>
</div>
<div class="section level2">
<h2 id="main-functions">Main functions<a class="anchor" aria-label="anchor" href="#main-functions"></a>
</h2>
<p>The main user-level functions of VplotR are <code><a href="reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code>, <code><a href="reference/plotVmat.html">plotVmat()</a></code>, <code><a href="reference/plotFootprint.html">plotFootprint()</a></code> and <code><a href="reference/plotProfile.html">plotProfile()</a></code>.</p>
<ul>
<li>
<code><a href="reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code> computes the distribution of fragment sizes over sets of genomic ranges;</li>
<li>
<code><a href="reference/plotVmat.html">plotVmat()</a></code> is used to compute fragment density and generate V-plots;</li>
<li>
<code><a href="reference/plotFootprint.html">plotFootprint()</a></code> generates the MNase-seq or ATAC-seq footprint at a set of genomic ranges.</li>
<li>
<code><a href="reference/plotProfile.html">plotProfile()</a></code> is used to plot the distribution of paired-end fragments at a single locus of interest.</li>
</ul>
<p>See the <a href="https://jserizay.com/VplotR/articles/VplotR.html" class="external-link">package vignette</a> for an in-depth introduction to the package functionalities and the <a href="https://jserizay.com/VplotR/reference/index.html" class="external-link">reference</a> for a full description of each function, a list of the available data provided in the package and examples of use cases.</p>
</div>
<div class="section level2">
<h2 id="getting-started-with-vplotr">Getting started with VplotR<a class="anchor" aria-label="anchor" href="#getting-started-with-vplotr"></a>
</h2>
<div class="section level3">
<h3 id="plotting-a-v-plot">Plotting a V-plot<a class="anchor" aria-label="anchor" href="#plotting-a-v-plot"></a>
</h3>
<p>V-plots can be generated using the <code><a href="reference/plotVmat.html">plotVmat()</a></code> function:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">MNase_sacCer3_Henikoff2011</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">ABF1_sacCer3</span><span class="op">)</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    x <span class="op">=</span> <span class="va">MNase_sacCer3_Henikoff2011</span>, </span>
<span>    granges <span class="op">=</span> <span class="va">ABF1_sacCer3</span></span>
<span><span class="op">)</span></span>
<span><span class="va">p</span></span></code></pre></div>
<div class="float">
<img src="inst/figures/vplot.png" alt="vplot"><div class="figcaption">vplot</div>
</div>
</div>
<div class="section level3">
<h3 id="importing-sequencing-fragments-as-granges">Importing sequencing fragments as <code>GRanges</code>
<a class="anchor" aria-label="anchor" href="#importing-sequencing-fragments-as-granges"></a>
</h3>
<p>Paired-end <code>.bam</code> files are imported using the <code><a href="reference/importPEBamFiles.html">importPEBamFiles()</a></code> function as follows:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/js2264/VplotR" class="external-link">VplotR</a></span><span class="op">)</span></span>
<span><span class="va">bamfile</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"ex1.bam"</span>, package <span class="op">=</span> <span class="st">"Rsamtools"</span><span class="op">)</span></span>
<span><span class="va">fragments</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/importPEBamFiles.html">importPEBamFiles</a></span><span class="op">(</span></span>
<span>    <span class="va">bamfile</span>,</span>
<span>    shift_ATAC_fragments <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span><span class="va">fragments</span></span></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">##   GRanges object with 1572 ranges and 0 metadata columns:</span></span>
<span><span class="co">##            seqnames    ranges strand</span></span>
<span><span class="co">##               &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">##        [1]     seq1    41-215      +</span></span>
<span><span class="co">##        [2]     seq1    54-255      +</span></span>
<span><span class="co">##        [3]     seq1    56-258      +</span></span>
<span><span class="co">##        [4]     seq1    65-255      +</span></span>
<span><span class="co">##        [5]     seq1    65-265      +</span></span>
<span><span class="co">##        ...      ...       ...    ...</span></span>
<span><span class="co">##     [1568]     seq2 1326-1542      -</span></span>
<span><span class="co">##     [1569]     seq2 1336-1544      -</span></span>
<span><span class="co">##     [1570]     seq2 1358-1550      -</span></span>
<span><span class="co">##     [1571]     seq2 1380-1557      -</span></span>
<span><span class="co">##     [1572]     seq2 1353-1562      -</span></span>
<span><span class="co">##     -------</span></span>
<span><span class="co">##     seqinfo: 2 sequences from an unspecified genome; no seqlengths</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="plotting-fragment-size-distribution">Plotting fragment size distribution<a class="anchor" aria-label="anchor" href="#plotting-fragment-size-distribution"></a>
</h3>
<p>The distribution of fragment sizes can be computed with the <code><a href="reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code> function:</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/js2264/VplotR" class="external-link">VplotR</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">MNase_sacCer3_Henikoff2011</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">ABF1_sacCer3</span><span class="op">)</span></span>
<span><span class="va">df</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/getFragmentsDistribution.html">getFragmentsDistribution</a></span><span class="op">(</span></span>
<span>    <span class="va">MNase_sacCer3_Henikoff2011</span>,</span>
<span>    <span class="va">ABF1_sacCer3</span></span>
<span><span class="op">)</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">df</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">x</span>, y <span class="op">=</span> <span class="va">y</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> </span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_path.html" class="external-link">geom_line</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> </span>
<span>    <span class="fu"><a href="reference/theme_ggplot2.html">theme_ggplot2</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> </span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Fragment size"</span>, y <span class="op">=</span> <span class="st">"# of fragments"</span><span class="op">)</span></span>
<span><span class="va">p</span></span></code></pre></div>
<div class="float">
<img src="inst/figures/fragment_size_distribution.png" alt="fragment_size_distribution"><div class="figcaption">fragment_size_distribution</div>
</div>
</div>
<div class="section level3">
<h3 id="datasets-provided-in-vplotr">Datasets provided in VplotR<a class="anchor" aria-label="anchor" href="#datasets-provided-in-vplotr"></a>
</h3>
<p>Several dataset samples are provided as <code>GRanges</code> object in this package:</p>
<ul>
<li>A set of tissue-specific ATAC-seq experiments in young adult <em>C. elegans</em> (Serizay et al., 2020):</li>
</ul>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/js2264/VplotR" class="external-link">VplotR</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">ATAC_ce11_Serizay2020</span><span class="op">)</span></span>
<span><span class="va">ATAC_ce11_Serizay2020</span></span></code></pre></div>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">##   $Germline</span></span>
<span><span class="co">##   GRanges object with 462371 ranges and 0 metadata columns:</span></span>
<span><span class="co">##              seqnames            ranges strand</span></span>
<span><span class="co">##                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">##          [1]     chrI           426-514      +</span></span>
<span><span class="co">##          [2]     chrI         3588-3854      +</span></span>
<span><span class="co">##          [3]     chrI         3640-3798      +</span></span>
<span><span class="co">##          [4]     chrI         3650-3694      +</span></span>
<span><span class="co">##          [5]     chrI         3732-3863      +</span></span>
<span><span class="co">##          ...      ...               ...    ...</span></span>
<span><span class="co">##     [462367]     chrX 17712277-17712469      -</span></span>
<span><span class="co">##     [462368]     chrX 17712279-17712342      -</span></span>
<span><span class="co">##     [462369]     chrX 17712282-17712565      -</span></span>
<span><span class="co">##     [462370]     chrX 17712285-17712384      -</span></span>
<span><span class="co">##     [462371]     chrX 17712287-17712576      -</span></span>
<span><span class="co">##     -------</span></span>
<span><span class="co">##     seqinfo: 7 sequences from an unspecified genome; no seqlengths</span></span>
<span><span class="co">##   </span></span>
<span><span class="co">##   $Neurons</span></span>
<span><span class="co">##   GRanges object with 367935 ranges and 0 metadata columns:</span></span>
<span><span class="co">##              seqnames            ranges strand</span></span>
<span><span class="co">##                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">##          [1]     chrI         4011-4241      +</span></span>
<span><span class="co">##          [2]     chrI         7397-7614      +</span></span>
<span><span class="co">##          [3]     chrI       11279-11502      +</span></span>
<span><span class="co">##          [4]     chrI       12744-12819      +</span></span>
<span><span class="co">##          [5]     chrI       14381-14433      +</span></span>
<span><span class="co">##          ...      ...               ...    ...</span></span>
<span><span class="co">##     [367931]     chrX 17687948-17687982      -</span></span>
<span><span class="co">##     [367932]     chrX 17699614-17699853      -</span></span>
<span><span class="co">##     [367933]     chrX 17706798-17706923      -</span></span>
<span><span class="co">##     [367934]     chrX 17708264-17708347      -</span></span>
<span><span class="co">##     [367935]     chrX 17709920-17710007      -</span></span>
<span><span class="co">##     -------</span></span>
<span><span class="co">##     seqinfo: 7 sequences from an unspecified genome; no seqlengths</span></span></code></pre></div>
<ul>
<li>An MNase-seq experiment in yeast (Henikoff et al., 2011) and associated ABF1 binding sites:</li>
</ul>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/js2264/VplotR" class="external-link">VplotR</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">MNase_sacCer3_Henikoff2011</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">##   GRanges object with 400000 ranges and 0 metadata columns:</span></span>
<span><span class="co">##              seqnames        ranges strand</span></span>
<span><span class="co">##                 &lt;Rle&gt;     &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">##          [1]     chrI         2-116      +</span></span>
<span><span class="co">##          [2]     chrI         14-66      +</span></span>
<span><span class="co">##          [3]     chrI        15-134      +</span></span>
<span><span class="co">##          [4]     chrI        54-167      +</span></span>
<span><span class="co">##          [5]     chrI        66-104      +</span></span>
<span><span class="co">##          ...      ...           ...    ...</span></span>
<span><span class="co">##     [399996]   chrXVI 920439-920471      -</span></span>
<span><span class="co">##     [399997]   chrXVI 920439-920486      -</span></span>
<span><span class="co">##     [399998]   chrXVI 920439-920528      -</span></span>
<span><span class="co">##     [399999]   chrXVI 920442-920659      -</span></span>
<span><span class="co">##     [400000]   chrXVI 920454-920683      -</span></span>
<span><span class="co">##     -------</span></span>
<span><span class="co">##     seqinfo: 17 sequences from an unspecified genome</span></span></code></pre></div>
<ul>
<li>A <code>.bam</code> file sample containing 100,000 reads mapped to <em>C. elegans</em>:</li>
</ul>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/js2264/VplotR" class="external-link">VplotR</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">bam_test</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">##   GRanges object with 100000 ranges and 0 metadata columns:</span></span>
<span><span class="co">##              seqnames        ranges strand</span></span>
<span><span class="co">##                 &lt;Rle&gt;     &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">##          [1]     chrI       425-554      +</span></span>
<span><span class="co">##          [2]     chrI       426-555      +</span></span>
<span><span class="co">##          [3]     chrI       459-604      +</span></span>
<span><span class="co">##          [4]     chrI       459-604      +</span></span>
<span><span class="co">##          [5]     chrI       504-634      +</span></span>
<span><span class="co">##          ...      ...           ...    ...</span></span>
<span><span class="co">##      [99996]     chrI 355817-355907      -</span></span>
<span><span class="co">##      [99997]     chrI 355863-356018      +</span></span>
<span><span class="co">##      [99998]     chrI 355864-355975      +</span></span>
<span><span class="co">##      [99999]     chrI 355880-355917      +</span></span>
<span><span class="co">##     [100000]     chrI 355883-355988      +</span></span>
<span><span class="co">##     -------</span></span>
<span><span class="co">##     seqinfo: 7 sequences from an unspecified genome; no seqlengths</span></span></code></pre></div>
<p>For a full list of the data available in this package, please go to the <a href="https://jserizay.com/VplotR/reference/index.html#section-data" class="external-link">reference page</a>.</p>
</div>
</div>
<div class="section level2">
<h2 id="contributions">Contributions<a class="anchor" aria-label="anchor" href="#contributions"></a>
</h2>
<p>Code contributions, bug reports, fixes and feature requests are most welcome. Please make any pull requests against the master branch at <a href="https://github.com/js2264/VplotR" class="external-link uri">https://github.com/js2264/VplotR</a> and file issues at <a href="https://github.com/js2264/VplotR/issues" class="external-link uri">https://github.com/js2264/VplotR/issues</a>.<br>
Feel free to reach out to <a href="mailto:jacquesserizay@gmail.com">J. Serizay</a> for any query.</p>
</div>
<div class="section level2">
<h2 id="license">License<a class="anchor" aria-label="anchor" href="#license"></a>
</h2>
<p><strong>VplotR</strong> is licensed under the GPL-3 license.</p>
</div>
<div class="section level2">
<h2 id="code-of-conduct">Code of Conduct<a class="anchor" aria-label="anchor" href="#code-of-conduct"></a>
</h2>
<p>Please note that <code>VplotR</code> is released with a <a href="https://contributor-covenant.org/version/2/0/CODE_OF_CONDUCT.html" class="external-link">Contributor Code of Conduct</a>.<br>
By contributing to this project, you agree to abide by its terms.</p>
</div>
</div>

  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2 data-toc-skip>Links</h2>
<ul class="list-unstyled">
<li><a href="https://www.bioconductor.org/packages/VplotR" class="external-link">View on Bioconductor</a></li>
<li><a href="https://github.com/js2264/VplotR/" class="external-link">Browse source code</a></li>
<li><a href="https://github.com/js2264/VplotR/issues" class="external-link">Report a bug</a></li>
<li><a href="https://js2264.github.io/" class="external-link">Contact the author at</a></li>
<li><a href="https://github.com/js2264/VplotR/" class="external-link">Browse source code at</a></li>
<li><a href="https://github.com/js2264/VplotR/blob/master/vignettes/VplotR.Rmd" class="external-link">Read vignette at</a></li>
</ul>
</div>

<div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li>GPL (&gt;= 3)</li>
</ul>
</div>

<div class="community">
<h2 data-toc-skip>Community</h2>
<ul class="list-unstyled">
<li><a href="CODE_OF_CONDUCT.html">Code of conduct</a></li>
</ul>
</div>

<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing VplotR</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>
<a href="https://js2264.github.io/" class="external-link">Jacques Serizay</a> <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0002-4295-0624" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>

<div class="dev-status">
<h2 data-toc-skip>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://doi.org/10.21105/joss.03009" class="external-link"><img src="https://joss.theoj.org/papers/10.21105/joss.03009/status.svg" alt="DOI"></a></li>
<li><a href="https://doi.org/10.1101/gr.265934.120" class="external-link"><img src="https://img.shields.io/badge/GR-%5BDOI%5D10.1101/gr.265934.120-green.svg" alt="DOI"></a></li>
<li><a href="https://www.bioconductor.org/packages/VplotR" class="external-link"><img src="https://img.shields.io/badge/release%20version-1.2.0-orange.svg"></a></li>
<li><a href="https://www.tidyverse.org/lifecycle/#production" class="external-link"><img src="https://img.shields.io/badge/lifecycle-production-blue.svg"></a></li>
<li><a href="https://github.com/js2264/VplotR" class="external-link"><img src="https://img.shields.io/github/languages/code-size/js2264/VplotR.svg"></a></li>
<li><a href="https://www.gnu.org/licenses/gpl-3.0.en.html" class="external-link"><img src="https://img.shields.io/badge/license-GPL--3-orange.svg"></a></li>
<li><a href="https://codecov.io/gh/js2264/VplotR?branch=master" class="external-link"><img src="https://codecov.io/gh/js2264/VplotR/branch/master/graph/badge.svg" alt="Codecov test coverage"></a></li>
<li><a href="https://github.com/js2264/VplotR/actions" class="external-link"><img src="https://github.com/js2264/VplotR/workflows/R-CMD-check/badge.svg" alt="R-CMD-check"></a></li>
</ul>
</div>

  </div>
</div>


      <footer><div class="copyright">
  <p></p>
<p>Developed by <a href="https://js2264.github.io/" class="external-link">Jacques Serizay</a>.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
