<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>VplotR â€¢ VplotR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/sandstone/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="VplotR">
<meta property="og:description" content="VplotR">
<meta property="og:image" content="/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">VplotR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.4.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/VplotR.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/js2264/VplotR/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>VplotR</h1>
                        <h4 class="author">Jacques Serizay</h4>
            
            <h4 class="date">2020-05-10</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/js2264/VplotR/blob/master/vignettes/VplotR.Rmd"><code>vignettes/VplotR.Rmd</code></a></small>
      <div class="hidden name"><code>VplotR.Rmd</code></div>

    </div>

    
    
<div id="importing-data" class="section level2">
<h2 class="hasAnchor">
<a href="#importing-data" class="anchor"></a>Importing data</h2>
<p>Paired-end .bam files are read using the <code><a href="../reference/importPEBamFiles.html">importPEBamFiles()</a></code> function as follows:</p>
<div class="sourceCode" id="cb1"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="no">VplotR</span>)
<span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="no">ce11_proms</span>)
<span class="no">ce11_proms</span>
<span class="no">fragments</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/importPEBamFiles.html">importPEBamFiles</a></span>(
    <span class="st">'ATAC-seq-mapped-fragments.bam'</span>,
    <span class="kw">where</span> <span class="kw">=</span> <span class="no">ce11_proms</span>,
    <span class="kw">shift_ATAC_fragments</span> <span class="kw">=</span> <span class="fl">TRUE</span>
)</pre></body></html></div>
<p><em>Note: to allow for a accurate background normalization, the <code>where</code> argument should be omitted.</em></p>
<p>Several datasets are available for this package:</p>
<ul>
<li>Sets of tissue-specific ATAC-seq experiments in young adult C. elegans:</li>
</ul>
<div class="sourceCode" id="cb2"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="no">ce11_proms</span>)
<span class="no">ce11_proms</span>
<span class="co">#&gt; GRanges object with 17458 ranges and 3 metadata columns:</span>
<span class="co">#&gt;                                   seqnames            ranges strand |</span>
<span class="co">#&gt;                                      &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; |</span>
<span class="co">#&gt;    nlp-40,homt-1_chrI_11273_11423     chrI       11273-11423      + |</span>
<span class="co">#&gt;    nlp-40,homt-1_chrI_11273_11423     chrI       11273-11423      - |</span>
<span class="co">#&gt;           rcor-1_chrI_26903_27053     chrI       26903-27053      - |</span>
<span class="co">#&gt;           sesn-1_chrI_36019_36169     chrI       36019-36169      - |</span>
<span class="co">#&gt;           sesn-1_chrI_42127_42277     chrI       42127-42277      - |</span>
<span class="co">#&gt;                               ...      ...               ...    ... .</span>
<span class="co">#&gt;    T23E7.2_chrX_17670496_17670646     chrX 17670496-17670646      + |</span>
<span class="co">#&gt;    T23E7.6_chrX_17684894_17685044     chrX 17684894-17685044      - |</span>
<span class="co">#&gt;    T23E7.6_chrX_17686030_17686180     chrX 17686030-17686180      - |</span>
<span class="co">#&gt;      cgt-2_chrX_17694789_17694939     chrX 17694789-17694939      + |</span>
<span class="co">#&gt;   H11L12.1_chrX_17711839_17711989     chrX 17711839-17711989      - |</span>
<span class="co">#&gt;                                     TSS.fwd   TSS.rev which.tissues</span>
<span class="co">#&gt;                                   &lt;numeric&gt; &lt;numeric&gt;      &lt;factor&gt;</span>
<span class="co">#&gt;    nlp-40,homt-1_chrI_11273_11423     11294     11416       Intest.</span>
<span class="co">#&gt;    nlp-40,homt-1_chrI_11273_11423     11294     11416       Intest.</span>
<span class="co">#&gt;           rcor-1_chrI_26903_27053     27038     26901         Ubiq.</span>
<span class="co">#&gt;           sesn-1_chrI_36019_36169     36112     36028       Intest.</span>
<span class="co">#&gt;           sesn-1_chrI_42127_42277     42216     42119          Soma</span>
<span class="co">#&gt;                               ...       ...       ...           ...</span>
<span class="co">#&gt;    T23E7.2_chrX_17670496_17670646  17670678  17670478        Muscle</span>
<span class="co">#&gt;    T23E7.6_chrX_17684894_17685044  17684919  17684932        Hypod.</span>
<span class="co">#&gt;    T23E7.6_chrX_17686030_17686180  17686189  17686064  Unclassified</span>
<span class="co">#&gt;      cgt-2_chrX_17694789_17694939  17694962  17694934       Intest.</span>
<span class="co">#&gt;   H11L12.1_chrX_17711839_17711989  17711974  17711854      Germline</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 6 sequences from an unspecified genome; no seqlengths</span>
<span class="no">ATAC_ce11</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/readRDS.html">readRDS</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/connections.html">url</a></span>(
    <span class="st">'https://ahringerlab.com/VplotR/ATAC_PE_fragments.rds'</span>
))
<span class="no">ATAC_ce11</span>
<span class="co">#&gt; $mixed</span>
<span class="co">#&gt; GRanges object with 24987916 ranges and 0 metadata columns:</span>
<span class="co">#&gt;              seqnames            ranges strand</span>
<span class="co">#&gt;                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;          [1]     chrI        9461-10168      +</span>
<span class="co">#&gt;          [2]     chrI        9723-10169      +</span>
<span class="co">#&gt;          [3]     chrI        9723-10353      +</span>
<span class="co">#&gt;          [4]     chrI        9723-10266      +</span>
<span class="co">#&gt;          [5]     chrI         9724-9880      +</span>
<span class="co">#&gt;          ...      ...               ...    ...</span>
<span class="co">#&gt;   [24987912]    chrIV 10254891-10255256      -</span>
<span class="co">#&gt;   [24987913]    chrIV 10254872-10255303      -</span>
<span class="co">#&gt;   [24987914]    chrIV 10254683-10255336      -</span>
<span class="co">#&gt;   [24987915]    chrIV 10254857-10255416      -</span>
<span class="co">#&gt;   [24987916]    chrIV 10254820-10255448      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $Germline</span>
<span class="co">#&gt; GRanges object with 11473757 ranges and 0 metadata columns:</span>
<span class="co">#&gt;              seqnames            ranges strand</span>
<span class="co">#&gt;                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;          [1]     chrI         9767-9863      +</span>
<span class="co">#&gt;          [2]     chrI         9767-9863      +</span>
<span class="co">#&gt;          [3]     chrI         9783-9932      +</span>
<span class="co">#&gt;          [4]     chrI         9791-9858      +</span>
<span class="co">#&gt;          [5]     chrI         9809-9996      +</span>
<span class="co">#&gt;          ...      ...               ...    ...</span>
<span class="co">#&gt;   [11473753]     chrX 17713403-17713666      -</span>
<span class="co">#&gt;   [11473754]     chrX 17713403-17713666      -</span>
<span class="co">#&gt;   [11473755]     chrX 17713386-17713731      -</span>
<span class="co">#&gt;   [11473756]     chrX 17713364-17713797      -</span>
<span class="co">#&gt;   [11473757]     chrX 17713364-17713797      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $Neurons</span>
<span class="co">#&gt; GRanges object with 24288112 ranges and 0 metadata columns:</span>
<span class="co">#&gt;              seqnames            ranges strand</span>
<span class="co">#&gt;                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;          [1]     chrI        9773-10171      +</span>
<span class="co">#&gt;          [2]     chrI         9780-9873      +</span>
<span class="co">#&gt;          [3]     chrI         9782-9936      +</span>
<span class="co">#&gt;          [4]     chrI        9782-10075      +</span>
<span class="co">#&gt;          [5]     chrI        9782-10075      +</span>
<span class="co">#&gt;          ...      ...               ...    ...</span>
<span class="co">#&gt;   [24288108]     chrX 17710929-17711148      -</span>
<span class="co">#&gt;   [24288109]     chrX 17710929-17711148      -</span>
<span class="co">#&gt;   [24288110]     chrX 17711000-17711149      -</span>
<span class="co">#&gt;   [24288111]     chrX 17711000-17711149      -</span>
<span class="co">#&gt;   [24288112]     chrX 17710805-17711154      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $Muscle</span>
<span class="co">#&gt; GRanges object with 9928267 ranges and 0 metadata columns:</span>
<span class="co">#&gt;             seqnames            ranges strand</span>
<span class="co">#&gt;                &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;         [1]     chrI         9764-9927      +</span>
<span class="co">#&gt;         [2]     chrI         9764-9927      +</span>
<span class="co">#&gt;         [3]     chrI         9773-9867      -</span>
<span class="co">#&gt;         [4]     chrI         9818-9879      -</span>
<span class="co">#&gt;         [5]     chrI         9844-9929      +</span>
<span class="co">#&gt;         ...      ...               ...    ...</span>
<span class="co">#&gt;   [9928263]     chrX 17711074-17711110      +</span>
<span class="co">#&gt;   [9928264]     chrX 17711083-17711141      +</span>
<span class="co">#&gt;   [9928265]     chrX 17710842-17711133      -</span>
<span class="co">#&gt;   [9928266]     chrX 17710842-17711133      -</span>
<span class="co">#&gt;   [9928267]     chrX 17711101-17711137      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $Hypod.</span>
<span class="co">#&gt; GRanges object with 15527998 ranges and 0 metadata columns:</span>
<span class="co">#&gt;              seqnames            ranges strand</span>
<span class="co">#&gt;                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;          [1]     chrI        9781-10119      +</span>
<span class="co">#&gt;          [2]     chrI        9781-10119      +</span>
<span class="co">#&gt;          [3]     chrI        9781-10119      +</span>
<span class="co">#&gt;          [4]     chrI         9865-9937      +</span>
<span class="co">#&gt;          [5]     chrI         9865-9937      +</span>
<span class="co">#&gt;          ...      ...               ...    ...</span>
<span class="co">#&gt;   [15527994]     chrX 17711078-17711142      -</span>
<span class="co">#&gt;   [15527995]     chrX 17711093-17711137      -</span>
<span class="co">#&gt;   [15527996]     chrX 17711083-17711156      -</span>
<span class="co">#&gt;   [15527997]     chrX 17711083-17711156      -</span>
<span class="co">#&gt;   [15527998]     chrX 17711083-17711156      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $Intest.</span>
<span class="co">#&gt; GRanges object with 24984513 ranges and 0 metadata columns:</span>
<span class="co">#&gt;              seqnames            ranges strand</span>
<span class="co">#&gt;                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;          [1]     chrI         9391-9858      +</span>
<span class="co">#&gt;          [2]     chrI         9474-9873      +</span>
<span class="co">#&gt;          [3]     chrI         9476-9936      +</span>
<span class="co">#&gt;          [4]     chrI         9476-9936      +</span>
<span class="co">#&gt;          [5]     chrI         9622-9936      +</span>
<span class="co">#&gt;          ...      ...               ...    ...</span>
<span class="co">#&gt;   [24984509]   chrIII 13768348-13768876      -</span>
<span class="co">#&gt;   [24984510]   chrIII 13768436-13768886      -</span>
<span class="co">#&gt;   [24984511]   chrIII 13768393-13768901      -</span>
<span class="co">#&gt;   [24984512]   chrIII 13768448-13768952      -</span>
<span class="co">#&gt;   [24984513]   chrIII 13768455-13768992      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome</span></pre></body></html></div>
<ul>
<li>MNase-seq experiment in yeast and REB1 binding sites:</li>
</ul>
<div class="sourceCode" id="cb3"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="no">REB1_sacCer3</span>)
<span class="no">REB1_sacCer3</span>
<span class="co">#&gt; GRanges object with 1264 ranges and 0 metadata columns:</span>
<span class="co">#&gt;          seqnames        ranges strand</span>
<span class="co">#&gt;             &lt;Rle&gt;     &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;      [1]     chrI   72794-72802      -</span>
<span class="co">#&gt;      [2]     chrI       118-126      +</span>
<span class="co">#&gt;      [3]     chrI       146-154      +</span>
<span class="co">#&gt;      [4]     chrI       128-136      +</span>
<span class="co">#&gt;      [5]     chrI   87164-87172      -</span>
<span class="co">#&gt;      ...      ...           ...    ...</span>
<span class="co">#&gt;   [1260]   chrXVI 413129-413137      +</span>
<span class="co">#&gt;   [1261]   chrXVI 942694-942702      -</span>
<span class="co">#&gt;   [1262]   chrXVI 360064-360072      -</span>
<span class="co">#&gt;   [1263]   chrXVI 743555-743563      -</span>
<span class="co">#&gt;   [1264]   chrXVI 627519-627527      +</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 16 sequences from an unspecified genome; no seqlengths</span>
<span class="no">MNase_sacCer3</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/readRDS.html">readRDS</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/connections.html">url</a></span>(
    <span class="st">'https://ahringerlab.com/VplotR/MNase_sacCer3_Henikoff2011.rds'</span>
))
<span class="no">MNase_sacCer3</span>
<span class="co">#&gt; GRanges object with 108397471 ranges and 0 metadata columns:</span>
<span class="co">#&gt;               seqnames        ranges strand</span>
<span class="co">#&gt;                  &lt;Rle&gt;     &lt;IRanges&gt;  &lt;Rle&gt;</span>
<span class="co">#&gt;           [1]     chrI         1-172      +</span>
<span class="co">#&gt;           [2]     chrI          1-89      +</span>
<span class="co">#&gt;           [3]     chrI          1-78      +</span>
<span class="co">#&gt;           [4]     chrI         1-167      +</span>
<span class="co">#&gt;           [5]     chrI         1-272      +</span>
<span class="co">#&gt;           ...      ...           ...    ...</span>
<span class="co">#&gt;   [108397467]   chrXVI 948004-948052      -</span>
<span class="co">#&gt;   [108397468]   chrXVI 947997-948052      -</span>
<span class="co">#&gt;   [108397469]   chrXVI 947991-948052      -</span>
<span class="co">#&gt;   [108397470]   chrXVI 947998-948052      -</span>
<span class="co">#&gt;   [108397471]   chrXVI 947997-948056      -</span>
<span class="co">#&gt;   -------</span>
<span class="co">#&gt;   seqinfo: 17 sequences from an unspecified genome</span></pre></body></html></div>
<ul>
<li>ATAC-seq in GM12878 human cells and CTCF binding sites:</li>
</ul>
<div class="sourceCode" id="cb4"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span>(<span class="no">CTCF_hg38</span>)
<span class="no">ATAC_hg38</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/readRDS.html">readRDS</a></span>(<span class="fu"><a href="https://rdrr.io/r/base/connections.html">url</a></span>(
    <span class="st">'https://ahringerlab.com/VplotR/ATAC_hg38_Chen2018.rds'</span>
))</pre></body></html></div>
</div>
<div id="fragment-size-distribution" class="section level2">
<h2 class="hasAnchor">
<a href="#fragment-size-distribution" class="anchor"></a>Fragment size distribution</h2>
<p>A preliminary control to check the distribution of fragment sizes (regardless of their location relative to genomic loci) can be performed using the <code><a href="../reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code> function.</p>
<div class="sourceCode" id="cb5"><html><body><pre class="r"><span class="no">df</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/getFragmentsDistribution.html">getFragmentsDistribution</a></span>(
    <span class="no">MNase_sacCer3</span>,
    <span class="no">REB1_sacCer3</span>
)
<span class="co">#&gt; Warning in as.cls(x): NAs introduced by coercion</span>

<span class="co">#&gt; Warning in as.cls(x): NAs introduced by coercion</span>

<span class="co">#&gt; Warning in as.cls(x): NAs introduced by coercion</span>
<span class="fu">ggplot</span>(<span class="no">df</span>, <span class="fu">aes</span>(<span class="kw">x</span> <span class="kw">=</span> <span class="no">x</span>, <span class="kw">y</span> <span class="kw">=</span> <span class="no">y</span>)) + <span class="fu">geom_line</span>()
<span class="co">#&gt; Warning: Removed 2 row(s) containing missing values (geom_path).</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-6-1.png" width="700"></p>
</div>
<div id="vplots" class="section level2">
<h2 class="hasAnchor">
<a href="#vplots" class="anchor"></a>Vplot(s)</h2>
<p>Once data is imported, a V-plot of paired-end fragments over loci of interest is generated using the <code><a href="../reference/plotVmat.html">plotVmat()</a></code> function:</p>
<div class="sourceCode" id="cb6"><html><body><pre class="r"><span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(<span class="no">MNase_sacCer3</span>, <span class="no">REB1_sacCer3</span>)
<span class="co">#&gt; Computing V-mat</span>
<span class="co">#&gt; Normalizing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-7-1.png" width="700"></p>
<p>The generation of multiple V-plots can be parallelized using a list of parameters:</p>
<div class="sourceCode" id="cb7"><html><body><pre class="r"><span class="no">list_params</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
    <span class="st">"MNase\n@ REB1"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(<span class="no">MNase_sacCer3</span>, <span class="no">REB1_sacCer3</span>),
    <span class="st">"MNase\n@ random loci"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(<span class="no">MNase_sacCer3</span>, <span class="fu"><a href="../reference/sampleGRanges.html">sampleGRanges</a></span>(<span class="no">REB1_sacCer3</span>))
)
<span class="no">p</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(
    <span class="no">list_params</span>,
    <span class="kw">cores</span> <span class="kw">=</span> <span class="fl">1</span>
)
<span class="co">#&gt; - Processing sample 1/2</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; - Processing sample 2/2</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="no">p</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-8-1.png" width="700"></p>
<div class="sourceCode" id="cb8"><html><body><pre class="r"><span class="co">#</span>
<span class="no">list_params</span> <span class="kw">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
    <span class="st">"Germline ATACseq\n@ Ubiq. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Germline'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Ubiq.'</span>]
    ),
    <span class="st">"Germline ATACseq\n@ Germline proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Germline'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Germline'</span>]
    ),
    <span class="st">"Neuron ATACseq\n@ Ubiq. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Neurons'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Ubiq.'</span>]
    ),
    <span class="st">"Neuron ATACseq\n@ Neuron proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Neurons'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Neurons'</span>]
    ),
    <span class="st">"Muscle ATACseq\n@ Ubiq. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Muscle'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Ubiq.'</span>]
    ),
    <span class="st">"Muscle ATACseq\n@ Muscle proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Muscle'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Muscle'</span>]
    ),
    <span class="st">"Hypod. ATACseq\n@ Ubiq. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Hypod.'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Ubiq.'</span>]
    ),
    <span class="st">"Hypod. ATACseq\n@ Hypod. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Hypod.'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Hypod.'</span>]
    ),
    <span class="st">"Intest. ATACseq\n@ Ubiq. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Intest.'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Ubiq.'</span>]
    ),
    <span class="st">"Intest. ATACseq\n@ Intest. proms"</span> <span class="kw">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html">list</a></span>(
        <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'Intest.'</span>]],
        <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">'Intest.'</span>]
    )
)
<span class="no">p</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(
    <span class="no">list_params</span>,
    <span class="kw">cores</span> <span class="kw">=</span> <span class="fl">1</span>,
    <span class="kw">nrow</span> <span class="kw">=</span> <span class="fl">2</span>, <span class="kw">ncol</span> <span class="kw">=</span> <span class="fl">5</span>,
    <span class="kw">verbose</span> <span class="kw">=</span> <span class="fl">FALSE</span>
)
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="no">p</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-8-2.png" width="700"></p>
</div>
<div id="vplots-normalization" class="section level2">
<h2 class="hasAnchor">
<a href="#vplots-normalization" class="anchor"></a>Vplots normalization</h2>
<p>Different normalization approaches are available:</p>
<div class="sourceCode" id="cb9"><html><body><pre class="r"><span class="co"># No normalization </span>
<span class="no">p</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(
    <span class="no">list_params</span>,
    <span class="kw">cores</span> <span class="kw">=</span> <span class="fl">1</span>,
    <span class="kw">nrow</span> <span class="kw">=</span> <span class="fl">2</span>, <span class="kw">ncol</span> <span class="kw">=</span> <span class="fl">5</span>,
    <span class="kw">verbose</span> <span class="kw">=</span> <span class="fl">FALSE</span>,
    <span class="kw">normFun</span> <span class="kw">=</span> <span class="st">''</span>
)
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="no">p</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-9-1.png" width="700"></p>
<div class="sourceCode" id="cb10"><html><body><pre class="r"><span class="co"># Library depth + number of loci of interest (default)</span>
<span class="no">p</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(
    <span class="no">list_params</span>,
    <span class="kw">cores</span> <span class="kw">=</span> <span class="fl">1</span>,
    <span class="kw">nrow</span> <span class="kw">=</span> <span class="fl">2</span>, <span class="kw">ncol</span> <span class="kw">=</span> <span class="fl">5</span>,
    <span class="kw">verbose</span> <span class="kw">=</span> <span class="fl">FALSE</span>,
    <span class="kw">normFun</span> <span class="kw">=</span> <span class="st">'libdepth+nloci'</span>
)
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Computing raw library depth</span>
<span class="co">#&gt; Dividing Vmat by its number of loci</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="no">p</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-9-2.png" width="700"></p>
<div class="sourceCode" id="cb11"><html><body><pre class="r"><span class="co"># Zscore</span>
<span class="no">p</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(
    <span class="no">list_params</span>,
    <span class="kw">cores</span> <span class="kw">=</span> <span class="fl">1</span>,
    <span class="kw">nrow</span> <span class="kw">=</span> <span class="fl">2</span>, <span class="kw">ncol</span> <span class="kw">=</span> <span class="fl">5</span>,
    <span class="kw">verbose</span> <span class="kw">=</span> <span class="fl">FALSE</span>,
    <span class="kw">normFun</span> <span class="kw">=</span> <span class="st">'zscore'</span>
)
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="no">p</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-9-3.png" width="700"></p>
<div class="sourceCode" id="cb12"><html><body><pre class="r"><span class="co"># Quantiles</span>
<span class="no">p</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span>(
    <span class="no">list_params</span>,
    <span class="kw">cores</span> <span class="kw">=</span> <span class="fl">1</span>,
    <span class="kw">nrow</span> <span class="kw">=</span> <span class="fl">2</span>, <span class="kw">ncol</span> <span class="kw">=</span> <span class="fl">5</span>,
    <span class="kw">verbose</span> <span class="kw">=</span> <span class="fl">FALSE</span>,
    <span class="kw">normFun</span> <span class="kw">=</span> <span class="st">'quantile'</span>,
    <span class="kw">s</span> <span class="kw">=</span> <span class="fl">0.99</span>
)
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="co">#&gt; Smoothing the matrix</span>
<span class="no">p</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-9-4.png" width="700"></p>
</div>
<div id="footprints" class="section level2">
<h2 class="hasAnchor">
<a href="#footprints" class="anchor"></a>Footprints</h2>
<p>VplotR also implements a function to profile the footprint from MNase or ATAC-seq over sets of genomic loci. For instance, CTCF is known for its ~40-bp large footprint at its binding loci.</p>
<div class="sourceCode" id="cb13"><html><body><pre class="r"><span class="fu"><a href="../reference/plotFootprint.html">plotFootprint</a></span>(
    <span class="no">ATAC_hg38</span>,
    <span class="no">CTCF_hg38</span>
)</pre></body></html></div>
<p><img src="https://raw.githubusercontent.com/js2264/VplotR/master/man/figures/CTCF_ATAC_hg38_Chen2018_footprint.png"></p>
</div>
<div id="nucleosome-enrichment-quantification" class="section level2">
<h2 class="hasAnchor">
<a href="#nucleosome-enrichment-quantification" class="anchor"></a>Nucleosome enrichment quantification</h2>
<p>Finally, the <code><a href="../reference/nucleosomeEnrichment.html">nucleosomeEnrichment()</a></code> function is useful to quantify and compare enrichment of flanking nucleosomes at promoters in ATAC-seq datasets.</p>
<div class="sourceCode" id="cb14"><html><body><pre class="r"><span class="no">germline_proms</span> <span class="kw">&lt;-</span> <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">"Germline"</span>]
<span class="no">muscle_proms</span> <span class="kw">&lt;-</span> <span class="no">ce11_proms</span>[<span class="no">ce11_proms</span>$<span class="no">which.tissues</span> <span class="kw">==</span> <span class="st">"Muscle"</span>]
<span class="co">#</span>
<span class="no">nuc_enrich_germline</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/nucleosomeEnrichment.html">nucleosomeEnrichment</a></span>(
    <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'mixed'</span>]],
    <span class="no">germline_proms</span>
)
<span class="co">#&gt; Computing Vmat...</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Computing background...</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Computing enrichment...</span>
<span class="no">nuc_enrich_germline</span>$<span class="no">fisher_test</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  Fisher's Exact Test for Count Data</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; data:  matrix(vec, ncol = 2)</span>
<span class="co">#&gt; p-value &lt; 2.2e-16</span>
<span class="co">#&gt; alternative hypothesis: true odds ratio is not equal to 1</span>
<span class="co">#&gt; 95 percent confidence interval:</span>
<span class="co">#&gt;  3.883553 4.096637</span>
<span class="co">#&gt; sample estimates:</span>
<span class="co">#&gt; odds ratio </span>
<span class="co">#&gt;   3.988432</span>
<span class="no">nuc_enrich_germline</span>$<span class="no">plot</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-11-1.png" width="700"></p>
<div class="sourceCode" id="cb15"><html><body><pre class="r"><span class="no">nuc_enrich_muscle</span> <span class="kw">&lt;-</span> <span class="fu"><a href="../reference/nucleosomeEnrichment.html">nucleosomeEnrichment</a></span>(
    <span class="no">ATAC_ce11</span><span class="kw">[[</span><span class="st">'mixed'</span>]],
    <span class="no">muscle_proms</span>
)
<span class="co">#&gt; Computing Vmat...</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Computing background...</span>
<span class="co">#&gt; No normalization applied</span>
<span class="co">#&gt; Computing enrichment...</span>
<span class="no">nuc_enrich_muscle</span>$<span class="no">fisher_test</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  Fisher's Exact Test for Count Data</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; data:  matrix(vec, ncol = 2)</span>
<span class="co">#&gt; p-value &lt; 2.2e-16</span>
<span class="co">#&gt; alternative hypothesis: true odds ratio is not equal to 1</span>
<span class="co">#&gt; 95 percent confidence interval:</span>
<span class="co">#&gt;  1.451124 1.524628</span>
<span class="co">#&gt; sample estimates:</span>
<span class="co">#&gt; odds ratio </span>
<span class="co">#&gt;   1.487388</span>
<span class="no">nuc_enrich_muscle</span>$<span class="no">plot</span></pre></body></html></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-11-2.png" width="700"></p>
<div id="session-info" class="section level3">
<h3 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session Info</h3>
<div class="sourceCode" id="cb16"><html><body><pre class="r"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span>()
<span class="co">#&gt; R version 3.5.2 (2018-12-20)</span>
<span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span>
<span class="co">#&gt; Running under: Ubuntu 18.04.2 LTS</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Matrix products: default</span>
<span class="co">#&gt; BLAS: /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.7.1</span>
<span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.7.1</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; locale:</span>
<span class="co">#&gt;  [1] LC_CTYPE=en_GB.UTF-8       LC_NUMERIC=C              </span>
<span class="co">#&gt;  [3] LC_TIME=en_GB.UTF-8        LC_COLLATE=en_GB.UTF-8    </span>
<span class="co">#&gt;  [5] LC_MONETARY=en_GB.UTF-8    LC_MESSAGES=en_GB.UTF-8   </span>
<span class="co">#&gt;  [7] LC_PAPER=en_GB.UTF-8       LC_NAME=C                 </span>
<span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span>
<span class="co">#&gt; [11] LC_MEASUREMENT=en_GB.UTF-8 LC_IDENTIFICATION=C       </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; attached base packages:</span>
<span class="co">#&gt; [1] parallel  stats4    stats     graphics  grDevices utils     datasets </span>
<span class="co">#&gt; [8] methods   base     </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; other attached packages:</span>
<span class="co">#&gt; [1] VplotR_0.4.2         magrittr_1.5         ggplot2_3.3.0       </span>
<span class="co">#&gt; [4] GenomicRanges_1.34.0 GenomeInfoDb_1.18.2  IRanges_2.16.0      </span>
<span class="co">#&gt; [7] S4Vectors_0.20.1     BiocGenerics_0.28.0 </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; loaded via a namespace (and not attached):</span>
<span class="co">#&gt;  [1] SummarizedExperiment_1.12.0 zoo_1.8-5                  </span>
<span class="co">#&gt;  [3] tidyselect_0.2.5            xfun_0.5                   </span>
<span class="co">#&gt;  [5] purrr_0.3.4                 reshape2_1.4.3             </span>
<span class="co">#&gt;  [7] lattice_0.20-38             colorspace_1.4-1           </span>
<span class="co">#&gt;  [9] htmltools_0.3.6             yaml_2.2.0                 </span>
<span class="co">#&gt; [11] rlang_0.4.6                 pkgdown_1.5.1              </span>
<span class="co">#&gt; [13] pillar_1.3.1                glue_1.4.0                 </span>
<span class="co">#&gt; [15] withr_2.2.0                 BiocParallel_1.16.6        </span>
<span class="co">#&gt; [17] RColorBrewer_1.1-2          plyr_1.8.4                 </span>
<span class="co">#&gt; [19] matrixStats_0.54.0          GenomeInfoDbData_1.2.0     </span>
<span class="co">#&gt; [21] stringr_1.4.0               zlibbioc_1.28.0            </span>
<span class="co">#&gt; [23] Biostrings_2.50.2           munsell_0.5.0              </span>
<span class="co">#&gt; [25] gtable_0.3.0                memoise_1.1.0              </span>
<span class="co">#&gt; [27] evaluate_0.13               labeling_0.3               </span>
<span class="co">#&gt; [29] Biobase_2.42.0              knitr_1.22                 </span>
<span class="co">#&gt; [31] Rcpp_1.0.1                  scales_1.0.0               </span>
<span class="co">#&gt; [33] backports_1.1.3             DelayedArray_0.8.0         </span>
<span class="co">#&gt; [35] desc_1.2.0                  XVector_0.22.0             </span>
<span class="co">#&gt; [37] fs_1.3.1                    Rsamtools_1.34.1           </span>
<span class="co">#&gt; [39] digest_0.6.18               stringi_1.3.1              </span>
<span class="co">#&gt; [41] dplyr_0.8.0.1               cowplot_0.9.4              </span>
<span class="co">#&gt; [43] grid_3.5.2                  rprojroot_1.3-2            </span>
<span class="co">#&gt; [45] tools_3.5.2                 bitops_1.0-6               </span>
<span class="co">#&gt; [47] RCurl_1.95-4.12             tibble_2.1.1               </span>
<span class="co">#&gt; [49] crayon_1.3.4                pkgconfig_2.0.2            </span>
<span class="co">#&gt; [51] MASS_7.3-51.1               Matrix_1.2-17              </span>
<span class="co">#&gt; [53] assertthat_0.2.1            rmarkdown_1.12.6           </span>
<span class="co">#&gt; [55] R6_2.4.0                    GenomicAlignments_1.18.1   </span>
<span class="co">#&gt; [57] compiler_3.5.2</span></pre></body></html></div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://js2264.github.io/">Jacques Serizay</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
