<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>VplotR • VplotR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/sandstone/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="VplotR">
<meta property="og:description" content="VplotR">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">VplotR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.12.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/VplotR.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/js2264/VplotR/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>VplotR</h1>
                        <h4 data-toc-skip class="author">Jacques
Serizay</h4>
            
            <h4 data-toc-skip class="date">2023-12-01</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/js2264/VplotR/blob/HEAD/vignettes/VplotR.Rmd" class="external-link"><code>vignettes/VplotR.Rmd</code></a></small>
      <div class="hidden name"><code>VplotR.Rmd</code></div>

    </div>

    
    
<ul>
<li>
<a href="#introduction">Introduction</a>
<ul>
<li><a href="#overview">Overview</a></li>
<li><a href="#installation">Installation</a></li>
</ul>
</li>
<li>
<a href="#importing-sequencing-datasets">Importing sequencing
datasets</a>
<ul>
<li><a href="#using-importpebamfiles-function">Using
<code>importPEBamFiles()</code> function</a></li>
<li><a href="#provided-datasets">Provided datasets</a></li>
</ul>
</li>
<li><a href="#fragment-size-distribution">Fragment size
distribution</a></li>
<li>
<a href="#vplots">Vplot(s)</a>
<ul>
<li><a href="#single-vplot">Single Vplot</a></li>
<li><a href="#multiple-vplots">Multiple Vplots</a></li>
<li><a href="#vplots-normalization">Vplots normalization</a></li>
</ul>
</li>
<li><a href="#footprints">Footprints</a></li>
<li><a href="#local-fragment-distribution">Local fragment
distribution</a></li>
<li><a href="#session-info">Session Info</a></li>
</ul>
<div class="section level2">
<h2 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<div class="section level3">
<h3 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h3>
<p>VplotR is an R package streamlining the process of generating
V-plots, i.e. two-dimensional paired-end fragment density plots. It
contains functions to import paired-end sequencing bam files from any
type of DNA accessibility experiments (e.g. ATAC-seq, DNA-seq,
MNase-seq) and can produce V-plots and one-dimensional footprint
profiles over single or aggregated genomic loci of interest. The R
package is well integrated within the Bioconductor environment and
easily fits in standard genomic analysis workflows. Integrating V-plots
into existing analytical frameworks has already brought additional
insights in chromatin organization (Serizay et al., 2020).</p>
<p>The main user-level functions of VplotR are
<code><a href="../reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code>, <code><a href="../reference/plotVmat.html">plotVmat()</a></code>,
<code><a href="../reference/plotFootprint.html">plotFootprint()</a></code> and <code><a href="../reference/plotProfile.html">plotProfile()</a></code>.</p>
<ul>
<li>
<code><a href="../reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code> computes the distribution of
fragment sizes over sets of genomic ranges;</li>
<li>
<code><a href="../reference/plotVmat.html">plotVmat()</a></code> is used to compute fragment density and
generate V-plots;</li>
<li>
<code><a href="../reference/plotFootprint.html">plotFootprint()</a></code> generates the MNase-seq or ATAC-seq
footprint at a set of genomic ranges.</li>
<li>
<code><a href="../reference/plotProfile.html">plotProfile()</a></code> is used to plot the distribution of
paired-end fragments at a single locus of interest.</li>
</ul>
</div>
<div class="section level3">
<h3 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h3>
<p>VplotR can be installed from Bioconductor:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span><span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"VplotR"</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://github.com/js2264/VplotR" class="external-link">"VplotR"</a></span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="importing-sequencing-datasets">Importing sequencing datasets<a class="anchor" aria-label="anchor" href="#importing-sequencing-datasets"></a>
</h2>
<div class="section level3">
<h3 id="using-importpebamfiles-function">Using <code>importPEBamFiles()</code> function<a class="anchor" aria-label="anchor" href="#using-importpebamfiles-function"></a>
</h3>
<p>Paired-end .bam files are imported using the
<code><a href="../reference/importPEBamFiles.html">importPEBamFiles()</a></code> function as follows:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/js2264/VplotR" class="external-link">VplotR</a></span><span class="op">)</span></span>
<span><span class="va">bamfile</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"ex1.bam"</span>, package <span class="op">=</span> <span class="st">"Rsamtools"</span><span class="op">)</span></span>
<span><span class="va">fragments</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importPEBamFiles.html">importPEBamFiles</a></span><span class="op">(</span></span>
<span>    <span class="va">bamfile</span>, </span>
<span>    shift_ATAC_fragments <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; &gt; Importing /__w/_temp/Library/Rsamtools/extdata/ex1.bam ...</span></span>
<span><span class="co">#&gt; &gt; Filtering /__w/_temp/Library/Rsamtools/extdata/ex1.bam ...</span></span>
<span><span class="co">#&gt; &gt; Shifting /__w/_temp/Library/Rsamtools/extdata/ex1.bam ...</span></span>
<span><span class="co">#&gt; &gt; /__w/_temp/Library/Rsamtools/extdata/ex1.bam import completed.</span></span>
<span><span class="va">fragments</span></span>
<span><span class="co">#&gt; GRanges object with 1572 ranges and 0 metadata columns:</span></span>
<span><span class="co">#&gt;          seqnames    ranges strand</span></span>
<span><span class="co">#&gt;             &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">#&gt;      [1]     seq1    41-215      +</span></span>
<span><span class="co">#&gt;      [2]     seq1    54-255      +</span></span>
<span><span class="co">#&gt;      [3]     seq1    56-258      +</span></span>
<span><span class="co">#&gt;      [4]     seq1    65-255      +</span></span>
<span><span class="co">#&gt;      [5]     seq1    65-265      +</span></span>
<span><span class="co">#&gt;      ...      ...       ...    ...</span></span>
<span><span class="co">#&gt;   [1568]     seq2 1326-1542      -</span></span>
<span><span class="co">#&gt;   [1569]     seq2 1336-1544      -</span></span>
<span><span class="co">#&gt;   [1570]     seq2 1358-1550      -</span></span>
<span><span class="co">#&gt;   [1571]     seq2 1380-1557      -</span></span>
<span><span class="co">#&gt;   [1572]     seq2 1353-1562      -</span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 2 sequences from an unspecified genome; no seqlengths</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="provided-datasets">Provided datasets<a class="anchor" aria-label="anchor" href="#provided-datasets"></a>
</h3>
<p>Several datasets are available for this package:</p>
<ul>
<li>Sets of tissue-specific ATAC-seq experiments in young adult C.
elegans (Serizay et al., 2020):</li>
</ul>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">ce11_proms</span><span class="op">)</span></span>
<span><span class="va">ce11_proms</span></span>
<span><span class="co">#&gt; GRanges object with 17458 ranges and 3 metadata columns:</span></span>
<span><span class="co">#&gt;           seqnames            ranges strand |   TSS.fwd   TSS.rev which.tissues</span></span>
<span><span class="co">#&gt;              &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt; &lt;numeric&gt;      &lt;factor&gt;</span></span>
<span><span class="co">#&gt;       [1]     chrI       11273-11423      + |     11294     11416       Intest.</span></span>
<span><span class="co">#&gt;       [2]     chrI       11273-11423      - |     11294     11416       Intest.</span></span>
<span><span class="co">#&gt;       [3]     chrI       26903-27053      - |     27038     26901       Ubiq.  </span></span>
<span><span class="co">#&gt;       [4]     chrI       36019-36169      - |     36112     36028       Intest.</span></span>
<span><span class="co">#&gt;       [5]     chrI       42127-42277      - |     42216     42119       Soma   </span></span>
<span><span class="co">#&gt;       ...      ...               ...    ... .       ...       ...           ...</span></span>
<span><span class="co">#&gt;   [17454]     chrX 17670496-17670646      + |  17670678  17670478  Muscle      </span></span>
<span><span class="co">#&gt;   [17455]     chrX 17684894-17685044      - |  17684919  17684932  Hypod.      </span></span>
<span><span class="co">#&gt;   [17456]     chrX 17686030-17686180      - |  17686189  17686064  Unclassified</span></span>
<span><span class="co">#&gt;   [17457]     chrX 17694789-17694939      + |  17694962  17694934  Intest.     </span></span>
<span><span class="co">#&gt;   [17458]     chrX 17711839-17711989      - |  17711974  17711854  Germline    </span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 6 sequences from an unspecified genome; no seqlengths</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">ATAC_ce11_Serizay2020</span><span class="op">)</span></span>
<span><span class="va">ATAC_ce11_Serizay2020</span></span>
<span><span class="co">#&gt; $Germline</span></span>
<span><span class="co">#&gt; GRanges object with 462371 ranges and 0 metadata columns:</span></span>
<span><span class="co">#&gt;            seqnames            ranges strand</span></span>
<span><span class="co">#&gt;               &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">#&gt;        [1]     chrI           426-514      +</span></span>
<span><span class="co">#&gt;        [2]     chrI         3588-3854      +</span></span>
<span><span class="co">#&gt;        [3]     chrI         3640-3798      +</span></span>
<span><span class="co">#&gt;        [4]     chrI         3650-3694      +</span></span>
<span><span class="co">#&gt;        [5]     chrI         3732-3863      +</span></span>
<span><span class="co">#&gt;        ...      ...               ...    ...</span></span>
<span><span class="co">#&gt;   [462367]     chrX 17712277-17712469      -</span></span>
<span><span class="co">#&gt;   [462368]     chrX 17712279-17712342      -</span></span>
<span><span class="co">#&gt;   [462369]     chrX 17712282-17712565      -</span></span>
<span><span class="co">#&gt;   [462370]     chrX 17712285-17712384      -</span></span>
<span><span class="co">#&gt;   [462371]     chrX 17712287-17712576      -</span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome; no seqlengths</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $Neurons</span></span>
<span><span class="co">#&gt; GRanges object with 367935 ranges and 0 metadata columns:</span></span>
<span><span class="co">#&gt;            seqnames            ranges strand</span></span>
<span><span class="co">#&gt;               &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">#&gt;        [1]     chrI         4011-4241      +</span></span>
<span><span class="co">#&gt;        [2]     chrI         7397-7614      +</span></span>
<span><span class="co">#&gt;        [3]     chrI       11279-11502      +</span></span>
<span><span class="co">#&gt;        [4]     chrI       12744-12819      +</span></span>
<span><span class="co">#&gt;        [5]     chrI       14381-14433      +</span></span>
<span><span class="co">#&gt;        ...      ...               ...    ...</span></span>
<span><span class="co">#&gt;   [367931]     chrX 17687948-17687982      -</span></span>
<span><span class="co">#&gt;   [367932]     chrX 17699614-17699853      -</span></span>
<span><span class="co">#&gt;   [367933]     chrX 17706798-17706923      -</span></span>
<span><span class="co">#&gt;   [367934]     chrX 17708264-17708347      -</span></span>
<span><span class="co">#&gt;   [367935]     chrX 17709920-17710007      -</span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 7 sequences from an unspecified genome; no seqlengths</span></span></code></pre></div>
<ul>
<li>MNase-seq experiment in yeast (Henikoff et al., 2011) and ABF1
binding sites:</li>
</ul>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">ABF1_sacCer3</span><span class="op">)</span></span>
<span><span class="va">ABF1_sacCer3</span></span>
<span><span class="co">#&gt; GRanges object with 567 ranges and 2 metadata columns:</span></span>
<span><span class="co">#&gt;         seqnames          ranges strand |  relScore       ID</span></span>
<span><span class="co">#&gt;            &lt;Rle&gt;       &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt;    &lt;Rle&gt;</span></span>
<span><span class="co">#&gt;     [1]    chrIV   392624-392639      + |  0.979866 MA0265.1</span></span>
<span><span class="co">#&gt;     [2]    chrIV 1196424-1196439      + |  0.979866 MA0265.1</span></span>
<span><span class="co">#&gt;     [3]   chrXIV   400687-400702      + |  0.979866 MA0265.1</span></span>
<span><span class="co">#&gt;     [4]    chrII   216540-216555      - |  0.978608 MA0265.1</span></span>
<span><span class="co">#&gt;     [5]   chrXVI     95317-95332      - |  0.974833 MA0265.1</span></span>
<span><span class="co">#&gt;     ...      ...             ...    ... .       ...      ...</span></span>
<span><span class="co">#&gt;   [563]    chrIV 1402786-1402801      + |  0.900182 MA0265.1</span></span>
<span><span class="co">#&gt;   [564]     chrX   545320-545335      + |  0.900182 MA0265.1</span></span>
<span><span class="co">#&gt;   [565]    chrXI   571301-571316      - |  0.900182 MA0265.1</span></span>
<span><span class="co">#&gt;   [566]   chrXIV   140631-140646      - |  0.900182 MA0265.1</span></span>
<span><span class="co">#&gt;   [567]   chrXVI   919179-919194      - |  0.900182 MA0265.1</span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 17 sequences from an unspecified genome; no seqlengths</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">MNase_sacCer3_Henikoff2011</span><span class="op">)</span></span>
<span><span class="va">MNase_sacCer3_Henikoff2011</span></span>
<span><span class="co">#&gt; GRanges object with 400000 ranges and 0 metadata columns:</span></span>
<span><span class="co">#&gt;            seqnames        ranges strand</span></span>
<span><span class="co">#&gt;               &lt;Rle&gt;     &lt;IRanges&gt;  &lt;Rle&gt;</span></span>
<span><span class="co">#&gt;        [1]     chrI         2-116      +</span></span>
<span><span class="co">#&gt;        [2]     chrI         14-66      +</span></span>
<span><span class="co">#&gt;        [3]     chrI        15-134      +</span></span>
<span><span class="co">#&gt;        [4]     chrI        54-167      +</span></span>
<span><span class="co">#&gt;        [5]     chrI        66-104      +</span></span>
<span><span class="co">#&gt;        ...      ...           ...    ...</span></span>
<span><span class="co">#&gt;   [399996]   chrXVI 920439-920471      -</span></span>
<span><span class="co">#&gt;   [399997]   chrXVI 920439-920486      -</span></span>
<span><span class="co">#&gt;   [399998]   chrXVI 920439-920528      -</span></span>
<span><span class="co">#&gt;   [399999]   chrXVI 920442-920659      -</span></span>
<span><span class="co">#&gt;   [400000]   chrXVI 920454-920683      -</span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 17 sequences from an unspecified genome</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="fragment-size-distribution">Fragment size distribution<a class="anchor" aria-label="anchor" href="#fragment-size-distribution"></a>
</h2>
<p>A preliminary control to check the distribution of fragment sizes
(regardless of their location relative to genomic loci) can be performed
using the <code><a href="../reference/getFragmentsDistribution.html">getFragmentsDistribution()</a></code> function.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getFragmentsDistribution.html">getFragmentsDistribution</a></span><span class="op">(</span></span>
<span>    <span class="va">MNase_sacCer3_Henikoff2011</span>, </span>
<span>    <span class="va">ABF1_sacCer3</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in as.cls(x): NAs introduced by coercion</span></span>
<span></span>
<span><span class="co">#&gt; Warning in as.cls(x): NAs introduced by coercion</span></span>
<span></span>
<span><span class="co">#&gt; Warning in as.cls(x): NAs introduced by coercion</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">df</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">x</span>, y <span class="op">=</span> <span class="va">y</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_path.html" class="external-link">geom_line</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> <span class="fu"><a href="../reference/theme_ggplot2.html">theme_ggplot2</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">p</span></span>
<span><span class="co">#&gt; Warning: Removed 2 rows containing missing values (`geom_line()`).</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-6-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="vplots">Vplot(s)<a class="anchor" aria-label="anchor" href="#vplots"></a>
</h2>
<div class="section level3">
<h3 id="single-vplot">Single Vplot<a class="anchor" aria-label="anchor" href="#single-vplot"></a>
</h3>
<p>Once data is imported, a V-plot of paired-end fragments over loci of
interest is generated using the <code><a href="../reference/plotVmat.html">plotVmat()</a></code> function:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">MNase_sacCer3_Henikoff2011</span>, granges <span class="op">=</span> <span class="va">ABF1_sacCer3</span><span class="op">)</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; No normalization applied</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-7-1.png" width="700"></p>
</div>
<div class="section level3">
<h3 id="multiple-vplots">Multiple Vplots<a class="anchor" aria-label="anchor" href="#multiple-vplots"></a>
</h3>
<p>The generation of multiple V-plots can be parallelized using a list
of parameters:</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">list_params</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    <span class="st">"MNase\n@ ABF1"</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="va">MNase_sacCer3_Henikoff2011</span>, <span class="va">ABF1_sacCer3</span><span class="op">)</span>, </span>
<span>    <span class="st">"MNase\n@ random loci"</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>        <span class="va">MNase_sacCer3_Henikoff2011</span>, <span class="fu"><a href="../reference/sampleGRanges.html">sampleGRanges</a></span><span class="op">(</span><span class="va">ABF1_sacCer3</span><span class="op">)</span></span>
<span>    <span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    <span class="va">list_params</span>, </span>
<span>    cores <span class="op">=</span> <span class="fl">1</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; - Processing sample 1/2</span></span>
<span><span class="co">#&gt; - Processing sample 2/2</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-8-1.png" width="700"></p>
<p>For instance, ATAC-seq fragment density can be visualized at
different classes of ubiquitous and tissue-specific promoters in <em>C.
elegans</em>.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">list_params</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    <span class="st">"Germline ATACseq\n@ Ubiq. proms"</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>        <span class="va">ATAC_ce11_Serizay2020</span><span class="op">[[</span><span class="st">'Germline'</span><span class="op">]</span><span class="op">]</span>, </span>
<span>        <span class="va">ce11_proms</span><span class="op">[</span><span class="va">ce11_proms</span><span class="op">$</span><span class="va">which.tissues</span> <span class="op">==</span> <span class="st">'Ubiq.'</span><span class="op">]</span></span>
<span>    <span class="op">)</span>, </span>
<span>    <span class="st">"Germline ATACseq\n@ Germline proms"</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>        <span class="va">ATAC_ce11_Serizay2020</span><span class="op">[[</span><span class="st">'Germline'</span><span class="op">]</span><span class="op">]</span>, </span>
<span>        <span class="va">ce11_proms</span><span class="op">[</span><span class="va">ce11_proms</span><span class="op">$</span><span class="va">which.tissues</span> <span class="op">==</span> <span class="st">'Germline'</span><span class="op">]</span></span>
<span>    <span class="op">)</span>,</span>
<span>    <span class="st">"Neuron ATACseq\n@ Ubiq. proms"</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>        <span class="va">ATAC_ce11_Serizay2020</span><span class="op">[[</span><span class="st">'Neurons'</span><span class="op">]</span><span class="op">]</span>, </span>
<span>        <span class="va">ce11_proms</span><span class="op">[</span><span class="va">ce11_proms</span><span class="op">$</span><span class="va">which.tissues</span> <span class="op">==</span> <span class="st">'Ubiq.'</span><span class="op">]</span></span>
<span>    <span class="op">)</span>, </span>
<span>    <span class="st">"Neuron ATACseq\n@ Neuron proms"</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>        <span class="va">ATAC_ce11_Serizay2020</span><span class="op">[[</span><span class="st">'Neurons'</span><span class="op">]</span><span class="op">]</span>, </span>
<span>        <span class="va">ce11_proms</span><span class="op">[</span><span class="va">ce11_proms</span><span class="op">$</span><span class="va">which.tissues</span> <span class="op">==</span> <span class="st">'Neurons'</span><span class="op">]</span></span>
<span>    <span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    <span class="va">list_params</span>, </span>
<span>    cores <span class="op">=</span> <span class="fl">1</span>,</span>
<span>    nrow <span class="op">=</span> <span class="fl">2</span>, ncol <span class="op">=</span> <span class="fl">5</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; - Processing sample 1/4</span></span>
<span><span class="co">#&gt; - Processing sample 2/4</span></span>
<span><span class="co">#&gt; - Processing sample 3/4</span></span>
<span><span class="co">#&gt; - Processing sample 4/4</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-9-1.png" width="700"></p>
</div>
<div class="section level3">
<h3 id="vplots-normalization">Vplots normalization<a class="anchor" aria-label="anchor" href="#vplots-normalization"></a>
</h3>
<p>Different normalization approaches are available using the
<code>normFun</code> argument.</p>
<ul>
<li>Un-normalized raw counts can be plotted by specifying
<code>normFun = 'none'</code>.</li>
</ul>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># No normalization </span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    <span class="va">list_params</span>, </span>
<span>    cores <span class="op">=</span> <span class="fl">1</span>, </span>
<span>    nrow <span class="op">=</span> <span class="fl">2</span>, ncol <span class="op">=</span> <span class="fl">5</span>, </span>
<span>    verbose <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    normFun <span class="op">=</span> <span class="st">'none'</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; No normalization applied</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; No normalization applied</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; No normalization applied</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; No normalization applied</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-10-1.png" width="700"></p>
<ul>
<li>By default, plots are normalized by the library depth of the
sequencing run and by the number of loci used to compute fragment
density.</li>
</ul>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Library depth + number of loci of interest (default)</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    <span class="va">list_params</span>, </span>
<span>    cores <span class="op">=</span> <span class="fl">1</span>, </span>
<span>    nrow <span class="op">=</span> <span class="fl">2</span>, ncol <span class="op">=</span> <span class="fl">5</span>, </span>
<span>    verbose <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    normFun <span class="op">=</span> <span class="st">'libdepth+nloci'</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Computing raw library depth</span></span>
<span><span class="co">#&gt; Dividing Vmat by its number of loci</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Computing raw library depth</span></span>
<span><span class="co">#&gt; Dividing Vmat by its number of loci</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Computing raw library depth</span></span>
<span><span class="co">#&gt; Dividing Vmat by its number of loci</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Computing raw library depth</span></span>
<span><span class="co">#&gt; Dividing Vmat by its number of loci</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-11-1.png" width="700"></p>
<ul>
<li>Alternatively, heatmaps can be internally z-scored or scaled to a
specific quantile.</li>
</ul>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Zscore</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    <span class="va">list_params</span>, </span>
<span>    cores <span class="op">=</span> <span class="fl">1</span>, </span>
<span>    nrow <span class="op">=</span> <span class="fl">2</span>, ncol <span class="op">=</span> <span class="fl">5</span>, </span>
<span>    verbose <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    normFun <span class="op">=</span> <span class="st">'zscore'</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-12-1.png" width="700"></p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Quantile</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotVmat.html">plotVmat</a></span><span class="op">(</span></span>
<span>    <span class="va">list_params</span>, </span>
<span>    cores <span class="op">=</span> <span class="fl">1</span>, </span>
<span>    nrow <span class="op">=</span> <span class="fl">2</span>, ncol <span class="op">=</span> <span class="fl">5</span>, </span>
<span>    verbose <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    normFun <span class="op">=</span> <span class="st">'quantile'</span>, </span>
<span>    s <span class="op">=</span> <span class="fl">0.99</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="co">#&gt; Computing V-mat</span></span>
<span><span class="co">#&gt; Normalizing the matrix</span></span>
<span><span class="co">#&gt; Smoothing the matrix</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-12-2.png" width="700"></p>
</div>
</div>
<div class="section level2">
<h2 id="footprints">Footprints<a class="anchor" aria-label="anchor" href="#footprints"></a>
</h2>
<p>VplotR also implements a function to profile the footprint from MNase
or ATAC-seq over sets of genomic loci. For instance, CTCF is known for
its ~40-bp large footprint at its binding loci.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotFootprint.html">plotFootprint</a></span><span class="op">(</span></span>
<span>    <span class="va">MNase_sacCer3_Henikoff2011</span>,</span>
<span>    <span class="va">ABF1_sacCer3</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; - Getting cuts</span></span>
<span><span class="co">#&gt; - Getting cut coverage</span></span>
<span><span class="co">#&gt; - Getting cut coverage / target</span></span>
<span><span class="co">#&gt; - Reformatting data into matrix</span></span>
<span><span class="co">#&gt; - Plotting footprint</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-13-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="local-fragment-distribution">Local fragment distribution<a class="anchor" aria-label="anchor" href="#local-fragment-distribution"></a>
</h2>
<p>VplotR provides a function to plot the distribution of paired-end
fragments over an individual genomic window.</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">MNase_sacCer3_Henikoff2011_subset</span><span class="op">)</span></span>
<span><span class="va">genes_sacCer3</span> <span class="op">&lt;-</span> <span class="fu">GenomicFeatures</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/GenomicFeatures/man/transcripts.html" class="external-link">genes</a></span><span class="op">(</span><span class="fu">TxDb.Scerevisiae.UCSC.sacCer3.sgdGene</span><span class="fu">::</span></span>
<span>    <span class="va"><a href="https://rdrr.io/pkg/TxDb.Scerevisiae.UCSC.sacCer3.sgdGene/man/package.html" class="external-link">TxDb.Scerevisiae.UCSC.sacCer3.sgdGene</a></span></span>
<span><span class="op">)</span></span>
<span><span class="va">p</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/plotProfile.html">plotProfile</a></span><span class="op">(</span></span>
<span>    fragments <span class="op">=</span> <span class="va">MNase_sacCer3_Henikoff2011_subset</span>,</span>
<span>    window <span class="op">=</span> <span class="st">"chrXV:186,400-187,400"</span>, </span>
<span>    loci <span class="op">=</span> <span class="va">ABF1_sacCer3</span>, </span>
<span>    annots <span class="op">=</span> <span class="va">genes_sacCer3</span>,</span>
<span>    min <span class="op">=</span> <span class="fl">20</span>, max <span class="op">=</span> <span class="fl">200</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, size <span class="op">=</span> <span class="fl">1.5</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Filtering and resizing fragments</span></span>
<span><span class="co">#&gt; 32276 fragments mapped over 1001 bases</span></span>
<span><span class="co">#&gt; Filtering and resizing fragments</span></span>
<span><span class="co">#&gt; Generating plot</span></span>
<span><span class="co">#&gt; Warning: Removed 49 rows containing missing values (`geom_line()`).</span></span>
<span><span class="co">#&gt; Warning: Removed 5176 rows containing missing values (`geom_point()`).</span></span>
<span><span class="co">#&gt; Warning: Removed 19 rows containing missing values (`geom_line()`).</span></span>
<span><span class="va">p</span></span></code></pre></div>
<p><img src="VplotR_files/figure-html/unnamed-chunk-14-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="session-info">Session Info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h2>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R Under development (unstable) (2023-11-22 r85609)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 22.04.3 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; time zone: UTC</span></span>
<span><span class="co">#&gt; tzcode source: system (glibc)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">#&gt; [8] base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt; [1] VplotR_1.12.1        ggplot2_3.4.4        GenomicRanges_1.55.1</span></span>
<span><span class="co">#&gt; [4] GenomeInfoDb_1.39.1  IRanges_2.37.0       S4Vectors_0.41.2    </span></span>
<span><span class="co">#&gt; [7] BiocGenerics_0.49.1 </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;   [1] DBI_1.1.3                                  </span></span>
<span><span class="co">#&gt;   [2] bitops_1.0-7                               </span></span>
<span><span class="co">#&gt;   [3] biomaRt_2.59.0                             </span></span>
<span><span class="co">#&gt;   [4] rlang_1.1.2                                </span></span>
<span><span class="co">#&gt;   [5] magrittr_2.0.3                             </span></span>
<span><span class="co">#&gt;   [6] matrixStats_1.1.0                          </span></span>
<span><span class="co">#&gt;   [7] compiler_4.4.0                             </span></span>
<span><span class="co">#&gt;   [8] RSQLite_2.3.3                              </span></span>
<span><span class="co">#&gt;   [9] GenomicFeatures_1.55.1                     </span></span>
<span><span class="co">#&gt;  [10] png_0.1-8                                  </span></span>
<span><span class="co">#&gt;  [11] systemfonts_1.0.5                          </span></span>
<span><span class="co">#&gt;  [12] vctrs_0.6.4                                </span></span>
<span><span class="co">#&gt;  [13] reshape2_1.4.4                             </span></span>
<span><span class="co">#&gt;  [14] stringr_1.5.1                              </span></span>
<span><span class="co">#&gt;  [15] pkgconfig_2.0.3                            </span></span>
<span><span class="co">#&gt;  [16] crayon_1.5.2                               </span></span>
<span><span class="co">#&gt;  [17] fastmap_1.1.1                              </span></span>
<span><span class="co">#&gt;  [18] dbplyr_2.4.0                               </span></span>
<span><span class="co">#&gt;  [19] XVector_0.43.0                             </span></span>
<span><span class="co">#&gt;  [20] labeling_0.4.3                             </span></span>
<span><span class="co">#&gt;  [21] utf8_1.2.4                                 </span></span>
<span><span class="co">#&gt;  [22] Rsamtools_2.19.2                           </span></span>
<span><span class="co">#&gt;  [23] rmarkdown_2.25                             </span></span>
<span><span class="co">#&gt;  [24] ragg_1.2.6                                 </span></span>
<span><span class="co">#&gt;  [25] purrr_1.0.2                                </span></span>
<span><span class="co">#&gt;  [26] bit_4.0.5                                  </span></span>
<span><span class="co">#&gt;  [27] xfun_0.41                                  </span></span>
<span><span class="co">#&gt;  [28] zlibbioc_1.49.0                            </span></span>
<span><span class="co">#&gt;  [29] cachem_1.0.8                               </span></span>
<span><span class="co">#&gt;  [30] jsonlite_1.8.7                             </span></span>
<span><span class="co">#&gt;  [31] progress_1.2.2                             </span></span>
<span><span class="co">#&gt;  [32] blob_1.2.4                                 </span></span>
<span><span class="co">#&gt;  [33] highr_0.10                                 </span></span>
<span><span class="co">#&gt;  [34] DelayedArray_0.29.0                        </span></span>
<span><span class="co">#&gt;  [35] BiocParallel_1.37.0                        </span></span>
<span><span class="co">#&gt;  [36] parallel_4.4.0                             </span></span>
<span><span class="co">#&gt;  [37] prettyunits_1.2.0                          </span></span>
<span><span class="co">#&gt;  [38] R6_2.5.1                                   </span></span>
<span><span class="co">#&gt;  [39] bslib_0.6.1                                </span></span>
<span><span class="co">#&gt;  [40] stringi_1.8.2                              </span></span>
<span><span class="co">#&gt;  [41] RColorBrewer_1.1-3                         </span></span>
<span><span class="co">#&gt;  [42] rtracklayer_1.63.0                         </span></span>
<span><span class="co">#&gt;  [43] jquerylib_0.1.4                            </span></span>
<span><span class="co">#&gt;  [44] Rcpp_1.0.11                                </span></span>
<span><span class="co">#&gt;  [45] SummarizedExperiment_1.33.1                </span></span>
<span><span class="co">#&gt;  [46] knitr_1.45                                 </span></span>
<span><span class="co">#&gt;  [47] zoo_1.8-12                                 </span></span>
<span><span class="co">#&gt;  [48] Matrix_1.6-3                               </span></span>
<span><span class="co">#&gt;  [49] tidyselect_1.2.0                           </span></span>
<span><span class="co">#&gt;  [50] abind_1.4-5                                </span></span>
<span><span class="co">#&gt;  [51] yaml_2.3.7                                 </span></span>
<span><span class="co">#&gt;  [52] codetools_0.2-19                           </span></span>
<span><span class="co">#&gt;  [53] curl_5.1.0                                 </span></span>
<span><span class="co">#&gt;  [54] lattice_0.22-5                             </span></span>
<span><span class="co">#&gt;  [55] tibble_3.2.1                               </span></span>
<span><span class="co">#&gt;  [56] plyr_1.8.9                                 </span></span>
<span><span class="co">#&gt;  [57] Biobase_2.63.0                             </span></span>
<span><span class="co">#&gt;  [58] withr_2.5.2                                </span></span>
<span><span class="co">#&gt;  [59] KEGGREST_1.43.0                            </span></span>
<span><span class="co">#&gt;  [60] evaluate_0.23                              </span></span>
<span><span class="co">#&gt;  [61] desc_1.4.2                                 </span></span>
<span><span class="co">#&gt;  [62] BiocFileCache_2.11.1                       </span></span>
<span><span class="co">#&gt;  [63] xml2_1.3.5                                 </span></span>
<span><span class="co">#&gt;  [64] Biostrings_2.71.1                          </span></span>
<span><span class="co">#&gt;  [65] filelock_1.0.2                             </span></span>
<span><span class="co">#&gt;  [66] pillar_1.9.0                               </span></span>
<span><span class="co">#&gt;  [67] MatrixGenerics_1.15.0                      </span></span>
<span><span class="co">#&gt;  [68] generics_0.1.3                             </span></span>
<span><span class="co">#&gt;  [69] rprojroot_2.0.4                            </span></span>
<span><span class="co">#&gt;  [70] RCurl_1.98-1.13                            </span></span>
<span><span class="co">#&gt;  [71] hms_1.1.3                                  </span></span>
<span><span class="co">#&gt;  [72] munsell_0.5.0                              </span></span>
<span><span class="co">#&gt;  [73] scales_1.3.0                               </span></span>
<span><span class="co">#&gt;  [74] glue_1.6.2                                 </span></span>
<span><span class="co">#&gt;  [75] tools_4.4.0                                </span></span>
<span><span class="co">#&gt;  [76] BiocIO_1.13.0                              </span></span>
<span><span class="co">#&gt;  [77] GenomicAlignments_1.39.0                   </span></span>
<span><span class="co">#&gt;  [78] fs_1.6.3                                   </span></span>
<span><span class="co">#&gt;  [79] XML_3.99-0.16                              </span></span>
<span><span class="co">#&gt;  [80] cowplot_1.1.1                              </span></span>
<span><span class="co">#&gt;  [81] grid_4.4.0                                 </span></span>
<span><span class="co">#&gt;  [82] TxDb.Scerevisiae.UCSC.sacCer3.sgdGene_3.2.2</span></span>
<span><span class="co">#&gt;  [83] AnnotationDbi_1.65.2                       </span></span>
<span><span class="co">#&gt;  [84] colorspace_2.1-0                           </span></span>
<span><span class="co">#&gt;  [85] GenomeInfoDbData_1.2.11                    </span></span>
<span><span class="co">#&gt;  [86] restfulr_0.0.15                            </span></span>
<span><span class="co">#&gt;  [87] cli_3.6.1                                  </span></span>
<span><span class="co">#&gt;  [88] rappdirs_0.3.3                             </span></span>
<span><span class="co">#&gt;  [89] textshaping_0.3.7                          </span></span>
<span><span class="co">#&gt;  [90] fansi_1.0.5                                </span></span>
<span><span class="co">#&gt;  [91] S4Arrays_1.3.1                             </span></span>
<span><span class="co">#&gt;  [92] dplyr_1.1.4                                </span></span>
<span><span class="co">#&gt;  [93] gtable_0.3.4                               </span></span>
<span><span class="co">#&gt;  [94] sass_0.4.7                                 </span></span>
<span><span class="co">#&gt;  [95] digest_0.6.33                              </span></span>
<span><span class="co">#&gt;  [96] SparseArray_1.3.1                          </span></span>
<span><span class="co">#&gt;  [97] rjson_0.2.21                               </span></span>
<span><span class="co">#&gt;  [98] farver_2.1.1                               </span></span>
<span><span class="co">#&gt;  [99] memoise_2.0.1                              </span></span>
<span><span class="co">#&gt; [100] htmltools_0.5.7                            </span></span>
<span><span class="co">#&gt; [101] pkgdown_2.0.7                              </span></span>
<span><span class="co">#&gt; [102] lifecycle_1.0.4                            </span></span>
<span><span class="co">#&gt; [103] httr_1.4.7                                 </span></span>
<span><span class="co">#&gt; [104] bit64_4.0.5</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by <a href="https://js2264.github.io/" class="external-link">Jacques Serizay</a>.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
